.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,41 @@
+name: Run Python Script and Commit Changes
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron:  '0 0 */5 * *' # Runs every 5 days at midnight
+  workflow_dispatch: # Allows manual triggering
+
+jobs:
+  run-and-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # fetch all history for all tags and branches
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8' # Set the Python version you need
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip GitPython packaging lxml emit
+          # Add any other dependencies here
+
+      - name: Run Python script
+        run: python scripts/run_param_parse.py
```

Comment:
jsom_from_xml also needs to run, the problem is that old firmwares failed on generate json files correctly.

Reply:
Where is json_from_xml being called ?

Reply:
I assumed we didnt need that anymore after the previous prs. I'll check this tomorrow